### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ravindra-rao**
merge develop into master

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+version: 2.1
+
+jobs:
```
Comment (ravindra-rao) [inline .circleci/config.yml:3]: job 1 is modified?

Comment (ravindra-rao) [reply .circleci/config.yml:3]: how many jobs are there?

**Review by ravindra-rao — COMMENTED** [review]
checked review and commenting

**Review by yodragon-github — COMMENTED** [review]
comment 1

**Review by yodragon-github — APPROVED** [review]

**Pull request merged**